src/styles/blocks/product.scss

```diff
@@ -0,0 +1,86 @@
+.product {
+  position: relative;
+  width: 200px;
+  height: 408px;
+  background-color: #fff;
+  border: 1px solid #f3f3f3;
+
+  display: flex;
+  flex-direction: column;
```

Comment:
👍 

But I'd also recommend you add horizontal padding - it's the same for all elements, so you can add it to card. It's optionally, but such an approach might help you to pass tests